crates/re_renderer/shader/generic_skybox.wgsl

```diff
@@ -39,15 +51,20 @@ fn dither_interleaved(rgb: vec3f, levels: f32, frag_coord: vec4<f32>) -> vec3f {
 @fragment
 fn main(in: FragmentInput, @builtin(position) frag_coord: vec4<f32>) -> @location(0) vec4f {
     let camera_dir = camera_ray_direction_from_screenuv(in.texcoord);
-    // Messing with direction a bit so it looks like in our old three-d based renderer (for easier comparison)
-    let rgb = skybox_dark_srgb(camera_dir); // TODO(andreas): Allow switching to skybox_light
+
+    var rgb: vec3f;
+    if uniforms.background_type == DIRECTIONAL_GRADIENT_DARK {
+        rgb = skybox_dark_srgb(camera_dir);
+    } else {
+        rgb = skybox_light_srgb(camera_dir);
+    }
 
     // Apply dithering in gamma space.
     // TODO(andreas): Once we switch to HDR outputs, this can be removed.
     //                As of writing, the render target itself is (s)RGB8, so we need to dither while we still have maximum precision.
-    let rgb_dithered = dither_interleaved(rgb, 256.0, frag_coord);
+    var rgb_linear_dithered = dither_interleaved(rgb, 256.0, frag_coord);
 
-    return vec4f(linear_from_srgb(rgb_dithered), 1.0);
+    return vec4f(linear_from_srgb(rgb_linear_dithered), 1.0);
```

Comment:
this naming seems backwards?

should be `rgb_gamma_dithered` and `linear_from_gamma(rgb_gamma_dithered)`